x/feegrant/simulation/decoder_test.go

```diff
@@ -22,6 +22,7 @@ var (
 )
 
 func TestDecodeStore(t *testing.T) {
+	t.Skip()
```

Comment:
Is there a reason this test was skipped?

Reply:
@sahith-narahari

Reply:
Oops sorry, this is unintended. Can you remove it in the store migration PR